e2e/lvm/lvm_suite_test.go

```diff
@@ -0,0 +1,18 @@
+package lvm_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
```

Comment:
It would be better to use ginkgo v2 from the start as that is the currently supported version. [See here](https://onsi.github.io/ginkgo/)

Reply:
Updated